content/docs/use-cases/versioning-data-and-model-files/tutorial.md

```diff
@@ -1,3 +1,10 @@
+---
+title: 'Tutorial: Data & Model Versioning'
```

Comment:
```suggestion
title: 'Data and Model Versioning Tutorial'
```
We should probably update the doc title and nav entry from & to "and".

Reply:
Same for "Sharing Data & Model Files" (nav entry)

Reply:
Unless "and" breaks the line (in the nav entries)

Reply:
For the meta title and description fields, we have to be careful about character length. In this case, the expanded title with the & is very close to the max length (<60 characters for title, <160 for description). Beyond that, Google will cut it off and append with an ellipsis. Here's an example with the new title mocked up:

<img width="626" alt="image" src="https://user-images.githubusercontent.com/18587991/96362906-734a7680-1139-11eb-9f72-02cc442d5041.png">

The meta title is the most important field SEO-wise, because it's actually checked for matching terms. Where it can be expanded with key terms (more likely for tutorials and use cases than command references), being economical with space helps to fit the needed terms. 

Yes, 'and' reads better, but it's okay to have both: the on page and navigation title can be 'and' while the meta title uses '&'.

Reply:
> the expanded title with the & is very close to the max length (<60 characters

As long as it's the last bit of  `Data Version Control · DVC` that gets truncated, it doesn't sound like a big deal to me. In any case, my suggestion is 1 char longer so should we commit it? (and apply to the doc's H1 title)

> while the meta title uses '&'.

Actually removing & from the nav in the use case rewrite anyway...

Reply:
\*Actually we're removing...

Reply:
> my suggestion is 1 char longer so should we commit it? (and apply to the doc's H1 title)

Applying this myself to move on. Thanks again @jeremydesroches !